app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> int | float:
+        pass
+
+    @abstractmethod
+    def player_info(self) -> int | float:

```

Comment:
The return type specified for the `player_info` method is incorrect. It should return a string.
Check `README.md` file

Reply:
since there are no division in those formulas, I agree (2)